### ISSUE TITLE
Contributing to `miunlock`

### ISSUE THREAD

**Issue by sskras** [Canny1913/miunlock#30]
I have new Poco F2 Pro at my hands. Can I contribute to the project?

At this time I:

* finished the MIUI setup (without Wi-Fi and without adding any accounts to the phone).
* enabled the USB debug mode on it.
* ensured that `adb` and `fastboot` commands work:
  
  ```
  $ fastboot oem device-info
  (bootloader) Verity mode: true
  (bootloader) Device unlocked: false
  (bootloader) Device critical unlocked: false
  (bootloader) Charger screen enabled: false
  OKAY [  0.016s]
  Finished. Total time: 0.028s
  
  $ fastboot getvar token
  token: VQEBHQEQ4lsEQmOVGmSJUBnjXyEcJQMDbG1pAgR4g_MU
  Finished. Total time: 0.007s
  
  $ fastboot getvar product
  product: lmi
  Finished. Total time: 0.003s
  ```

Is the project interested in getting dumps of HTTP or USB communication collected during the unlocking process?

**Comment by sskras** [comment]
And if so, is there any tutorials / configurations known to work best?

**Comment by Canny1913** [comment]
yes i would love a wireshark dump but only the dump of official miunlock tool.

**Comment by sskras** [comment]
Will try to do.

Any HOWTOs around about setting up Wireshark to enable the TLS decryption?
Or does the official tool work via unencrypted, plain HTTP?

**Comment by Canny1913** [comment]
i actually have no clue since ive never did that
i dont think official tool is encrypted tho

**Comment by cronerstat** [comment]
Hi I have a bricked Redmi K60 with locked bootloader and the system says it is destroyed, do this tool help to unlock the bootloader and flash using MiFlash?

**Comment by cronerstat** [comment]
[](url)
![error](https://github.com/Canny1913/miunlock/assets/36487463/7cf0215f-7bb5-40c1-9e8e-c7a558bd4d2d)
 that is the error that I get when trying to verify the account when using the main.py

**Comment by Canny1913** [comment]
> ![error](https://user-images.githubusercontent.com/36487463/265181606-7cf0215f-7bb5-40c1-9e8e-c7a558bd4d2d.jpg) that is the error that I get when trying to verify the account when using the main.py

that means the sms verification was successful, re-do what the program tells you to do

**Comment by Canny1913** [comment]
> Hi I have a bricked Redmi K60 with locked bootloader and the system says it is destroyed, do this tool help to unlock the bootloader and flash using MiFlash?

if you have already unlocked before and have access to fastboot, you should be able to unlock again and rescue your device

**Comment by cronerstat** [comment]
> > ![error](https://user-images.githubusercontent.com/36487463/265181606-7cf0215f-7bb5-40c1-9e8e-c7a558bd4d2d.jpg) that is the error that I get when trying to verify the account when using the main.py
> 
> that means the sms verification was successful, re-do what the program tells you to do

But I tried 4 times and still was showing the same error message asking to verify my Xiaomi account since I needed to exe the main.py again

**Comment by sskras** [comment]
@cronerstat, I think you should rise a new issue.

(New topic in Discussion could be even more appropriate place for analyzding your situation, but the Discussion tab is not enabled on this repo. Hint @Canny1913:)

**Comment by Canny1913** [comment]
> > > ![error](https://user-images.githubusercontent.com/36487463/265181606-7cf0215f-7bb5-40c1-9e8e-c7a558bd4d2d.jpg) that is the error that I get when trying to verify the account when using the main.py
> > 
> > 
> > that means the sms verification was successful, re-do what the program tells you to do
> 
> But I tried 4 times and still was showing the same error message asking to verify my Xiaomi account since I needed to exe the main.py again

im experiencing the same issue with the new update im currently working on, dont really know whats causing it at the moment
